List-of-Free-Code-Camp-city-based-Campsites.md

```diff
@@ -564,6 +566,7 @@ If your city isn't listed here, you should [[create a Campsite for your city|How
 - Madagascar
   - [Antananarivo](https://www.facebook.com/groups/free.code.camp.antananarivo/)
 - Malaysia
+  - [Ipoh](https://www.facebook.com/groups/free.code.camp.your.perak/)
```

Comment:
We give them clear and straightforward instructions to make URLs based on their cities... they simply give me the URL after the have created them... in cases like these in which they make them based on their state, I tell them to choose the city for the title and the description.
This specific one chose not even Ipoh but something 12kms away. I'm ok with that. People don't really care for the URLS anyway. LOL